topo/node/arista/arista.go

```diff
@@ -359,6 +359,12 @@ func defaults(pb *tpb.Node) *tpb.Node {
 			},
 		}
 	}
+	if pb.Os == "" {
```

Comment:
because arista uses a controller, are these values reflected in the final topo proto?

ideally we would leave defaults to them, but we also need the defaulted info to appear in the final textproto so we should prioritize that over controller defaults